.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,7 +2,4 @@ Closes #
 
 ### Description
 
-### Definition of Done
-* [ ] The change has a corresponding task. Each task needs to be well defined and described.
-* [ ] Documentation (README, comments, etc.) is updated accordingly.
-* [ ] Change has been tested (manually or with automated tests).
+### How to test
```

Comment:
Not sure what this change does

Reply:
It replaces a DoD checklist in the PR with a paragraph to fill which I find more useful :slightly_smiling_face: